### PR TITLE
Merge API Reference and Help Center integration tests

### PR DESCRIPTION
**Pull request by pedrottimark**
## Description

Prerequisite to replace backend redirect for /docs/product with frontend link to OpenShift docs page.

### Changed files

1. Delete cypress/constants/Docs.js

2. Rename cypress/integration/apiReference.test.js as main/HelpMenu.test.js [too many changes to compare side-by-side]
    * Add `describe` block for Help Center.
    * Add `describe` block for version number.

3. Delete cypress/integration/docs.test.js

4. Edit src/Containers/MainPage/Header/HelpMenu.tsx
    * Merge items into one `ApplicationLauncherGroup` element to prepare for conditional rendering of Help Center, separator, and version number.

### Residue

1. Move files from credentialExpiry folder into main folder.
2. Add tests for other banners.

## Checklist
- [x] Investigated and inspected CI test results
- [x] Edited integration tests

## Testing Performed

Ran tests individually with local deployment before and after editing.

Help menu has same appearance after merging two `ul` into one.
Help_menu

### CODE REVIEW

File: ui/apps/platform/cypress/integration/main/help.test.js [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+    it('should be a disabled link', () => {
+        visitMainDashboard();
+
+        /*
+         * Open Help menu, and then wait for an authenticated metadata request,
+         * which is prerequisite to render the version number below.
+         */
+        interactAndWaitForResponses(() => {
+            cy.get('button[aria-label="Help menu"').click();
```
Comment (0x656b694d) [inline ui/apps/platform/cypress/integration/main/help.test.js:77]:
```suggestion
            cy.get('button[aria-label="Help menu"]').click();
```

**Review by 0x656b694d — CHANGES_REQUESTED** [review]

File: ui/apps/platform/cypress/integration/main/help.test.js (hunk 1 above)
Comment (pedrottimark) [reply ui/apps/platform/cypress/integration/main/help.test.js:77]: Good eyes!

**Review by bradr5 — APPROVED** [review]
Tests make sense to me 👍

**Review by 0x656b694d — APPROVED** [review]
LGTM

**Pull request merged**